src/App.js

```diff
@@ -1,23 +1,63 @@
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from "react";
+import "./App.css";
+import Footer from "./components/footer/Footer";
+import NavigationBar from "./components/header/NavigationBar";
+import Cards from "./components/main/Cards";
+import SearchBar from "./components/main/SearchBar";
+import UserInformation from "./components/main/FolderInformation";
+import { getCards, getUsers } from "./services/api";
 
 function App() {
+  // 카드 정보 받아오기
+  const [folderOwnerData, setFolderOwnerData] = useState([]);
+  const [folderNameData, setFolderNameData] = useState();
+  const [cardData, setCardData] = useState([]);
```

Comment:
1. 상태명에는 -Data 라는 접미사는 잘쓰지 않는 편입니다! (어차피 useState로 선언한것 자체가 데이터라는걸 의미하니까요)
2. folderOwnerData랑 cardData는 데이터타입이 배열인데, 상태명만 봤을때는 배열임을 인지할 수 없는 것 같아요! 배열인 변수명에는 -s 접미사를 붙이거나 -list 접미사를 붙여서 데이터타입이 배열임을 나타내주는게 좋아요!

```suggestion
  const [folderOwners, setFolderOwners] = useState([]);
  const [folderName, setFolderName] = useState();
  const [cards, setCards] = useState([]);
```